Makefile

```diff
@@ -672,6 +673,7 @@ OBJS += libs/json11/json11.o
 
 OBJS += libs/ezsat/ezsat.o
 OBJS += libs/ezsat/ezminisat.o
+OBJS += libs/ezsat/ezcommand.o
```

Comment:
This can be a bit of bikeshedding but I suggest `ezcmdline` or `ezextern` for the name.

Reply:
Fine by me. ezcmdlinesat for the file and ezCmdLineSAT for the class to keep it consistent with the rest?

Reply:
I renamed as described above

Reply:
Sorry to bikeshed but I think it's better to have `ezcmdline`, since the fact that it's an interface to a SAT solver is clear from the context, and `ezCmdlineSAT` since cmdline is a single word as used